common/src/main/resources/common/common.bom

```diff
@@ -172,3 +172,52 @@ brooklyn.catalog:
           launch.command: |
             haproxy -f haproxy.conf -f servers.conf -c &&
               haproxy -D -f haproxy.conf -f servers.conf -sf $(cat ${PID_FILE})
+
+        brooklyn.initializers:
+        - type: org.apache.brooklyn.core.effector.ssh.SshCommandEffector
+          brooklyn.config:
+            name: configureHaproxy
```

Comment:
Can we change the effector name to just `configureNodes` to prevent ambiguity about capitalisation of HAProxy in the name?